src/pages/VolunteersFind.page

```diff
@@ -127,25 +120,29 @@
                     </div>
                 </div>
                 <div class="slds-media__body">
-                    <div class="slds-text-title--caps">Choose Criteria</div>
+                    <div class="slds-text-title--caps">{!$Label.labelChooseCriteria}</div>
                 </div>
             </div>
             <div>
                 <apex:outputPanel layout="block" style="font-style:italic;" styleClass="slds-m-around--small">{!$Label.labelFindVolunteersHelpCriteria}</apex:outputPanel> 
                 <form class="slds-form--stacked">
                     <div class="slds-grid">
-                        <apex:repeat value="{!$ObjectType.Contact.FieldSets.VolunteersFindCriteriaFS}" var="field">
+                        <apex:repeat value="{!$ObjectType.Contact.FieldSets.VolunteersFindCriteriaFS}" var="f">
                             <div class="slds-size--1-of-{!$ObjectType.Contact.FieldSets.VolunteersFindCriteriaFS.size}">
                                 <div class="slds-form-element slds-p-around--x-small">
-                                    <apex:outputLabel value="{!field.Label}" for="idInputField" styleClass="slds-form-element__label" />
+                                    <apex:outputLabel for="idInputField" styleClass="slds-form-element__label">
+                                        <span class="required slds-required">{!IF(f.DBRequired || f.Required, '*', '')}</span>{!f.Label}
```

Comment:
we don't actually need the required identifier here, since in the criteria section, we don't want to force them to fill out all criteria in the field set.